sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_utils.py

```diff
@@ -328,3 +328,29 @@ def set_event_loop_policy() -> None:
         # Reference: https://stackoverflow.com/questions/45600579/asyncio-event-loop-is-closed-when-getting-loop
         # On Windows seems to be a problem with EventLoopPolicy, use this snippet to work around it
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())  # type: ignore[attr-defined]
+
+
+class JSONLDataFileLoader:
+    def __init__(self, filename: Union[os.PathLike, str]):
+        self.filename = filename
+
+    def load(self) -> pd.DataFrame:
+        return pd.read_json(self.filename, lines=True)
+
+
+class CSVDataFileLoader:
+    def __init__(self, filename: Union[os.PathLike, str]):
+        self.filename = filename
+
+    def load(self) -> pd.DataFrame:
+        return pd.read_csv(self.filename)
```

Comment:
Would this line throw any error if the csv doesn't come with the header row?

Reply:
Yep! The behavior is the same as the current behavior now if a JSONL file is missing one of the necessary input fields (e.g. "response", "ground_truth", etc.). The command will fail with `(UserError) <EvaluatorName>: Either 'conversation' or individual inputs must be provided.`

I've added a unittest to demonstrate this.